lesson_71/HW_L71/models/user.js

```diff
@@ -0,0 +1,32 @@
+import mongoose from "mongoose";
+
+const Schema = mongoose.Schema;
+
+const userSchema = new Schema({
+  email: {
+    type: String,
+    required: true,
+    max: 254,
+    validate: {
+      validator: async function(email) {
+        const user = await this.constructor.findOne({ email });
+        return !user;
+      },
+      message: "user already exists"
+    }
+  },
+  password: {
```

Comment:
you can use `match` with RegExp to validate password and email 
[https://mongoosejs.com/docs/schematypes.html#string-validators](https://mongoosejs.com/docs/schematypes.html#string-validators)